WooCommerce/StoreWidgets/StoreInfoProvider.swift

```diff
@@ -39,35 +39,33 @@ struct StoreInfoProvider: TimelineProvider {
     /// Redacted entry with sample data.
     ///
     func placeholder(in context: Context) -> StoreInfoEntry {
-        StoreInfoEntry(date: Date(),
-                       range: "Today",
-                       name: "Ernest Shop",
-                       revenue: "$132.234",
-                       visitors: "67",
-                       orders: "23",
-                       conversion: "37%")
+        let dependencies = Self.fetchDependencies()
+        return StoreInfoEntry(date: Date(),
+                              range: "Today",
+                              name: dependencies?.storeName ?? Localization.myShop,
+                              revenue: "$132.234",
+                              visitors: "67",
+                              orders: "23",
+                              conversion: "37%")
```

Comment:
23/67 = 0,34328358, am I missing something?

Reply:
Oh, I never paid attention to those values, just wrote random numbers, but better for them to make sense! Good catch